python/lsst/ip/diffim/dcrModel.py

```diff
@@ -406,34 +418,36 @@ def clampModel(self, subfilter, newModel, bbox, statsCtrl, regularizeSigma, mode
             Sub-region to coadd
         statsCtrl : `lsst.afw.math.StatisticsControl`
             Statistics control object for coadd
-        regularizeSigma : `float`
-            Threshold to exclude noise-like pixels from regularization.
-        modelClampFactor : `float`
+        clamp : `float`
             Maximum relative change of the model allowed between iterations.
+        sigma : `float`
+            Threshold to exclude noise-like pixels from regularization.
+        bufferSize : `int`
+            Number of additional pixels to exclude
+            from the edges of the bounding box.
         convergenceMaskPlanes : `list` of `str`, or `str`, optional
             Mask planes to use to calculate convergence.
-            Default value is set in ``calculateNoiseCutoff`` if not supplied.
+        regularizationWidth : int, optional
+            Minimum radius of a region to include in regularization, in pixels.
         """
+        denoiseStructure = ndimage.iterate_structure(ndimage.generate_binary_structure(2, 1),
```

Comment:
I'll move this code to a separate function, and try to make the usage clear in the comments there. 
This does smooth out the edges, but only of region of pixels being selected. The smoothing actually improves the overall results, since the artifacts should not have ragged edges.